atomic_reactor/constants.py

```diff
@@ -14,6 +14,77 @@
 BUILD_JSON_ENV = 'BUILD_JSON'
 RESULTS_JSON = 'results.json'
 
+JSON_SCHEMA = {
```

Comment:
It is validating ATOMIC_REACTOR_PLUGINS env var (its just being called 'build JSON' internally), the one we generate from `prod_inner.json`.